src/client/keycode.cpp

```diff
@@ -99,37 +99,37 @@ static const struct table_key table[] = {
 	// Keys without a Char
 	DEFINEKEY1(KEY_LBUTTON, N_("Left Button"))
 	DEFINEKEY1(KEY_RBUTTON, N_("Right Button"))
-	DEFINEKEY1(KEY_CANCEL, N_("Cancel"))
+	DEFINEKEY1(KEY_CANCEL, N_("Break")) // Usually paired with the Pause key
 	DEFINEKEY1(KEY_MBUTTON, N_("Middle Button"))
 	DEFINEKEY1(KEY_XBUTTON1, N_("X Button 1"))
 	DEFINEKEY1(KEY_XBUTTON2, N_("X Button 2"))
 	DEFINEKEY1(KEY_BACK, N_("Backspace"))
 	DEFINEKEY1(KEY_TAB, N_("Tab"))
-	DEFINEKEY1(KEY_CLEAR, N_("Clear"))
+	DEFINEKEY1(KEY_CLEAR, N_("Clear Key"))
 	DEFINEKEY1(KEY_RETURN, N_("Return"))
 	DEFINEKEY1(KEY_SHIFT, N_("Shift"))
 	DEFINEKEY1(KEY_CONTROL, N_("Control"))
```

Comment:
there might be no overlap currently but these three are also at risk since their text is so generic

Reply:
I agree. Should it be ended with "Key" (like others), too?

Reply:
the ideal solution would be if gettext allowed us to differentiate translations with the same text but different meaning/context.
but it doesn't so yeah, "Key" works.